### PR TITLE
Changed Node Default State to LOW 

### PR DESCRIPTION
**Pull request by git-akshay**
Signed Off by : Akshay kumar workspace.akshay.kumar@gmail.com

Description: Solves Issue  #14
Changed the default node state to low (updated the setup function in src/ESP8266.cpp).
Patch file attached.

TEST: Not tested. Verification needed

KNOWN LIMITATIONS: None

### CODE REVIEW

**Comment by iayanpahwa** [comment]
@git-akshay we dont support direct patch file integration in our current build system as of now, please remove it.

**Comment by git-akshay** [comment]
@iayanpahwa Removed the patch file but could not perform any tests yet. :/

**Comment by iayanpahwa** [comment]
@git-akshay , I've tested at my end, somehow the issue still persists, The D0 pin seems unmaskable with LOW state, please investigate further and make necessary changes. Holding this PR for time.

File: src/ESP8266.cpp [hunk 1]
```diff
@@ -41,7 +41,7 @@ void setup() {
   #endif
 
   pinMode(RELAY,OUTPUT); //Configure Pin as Output
-
+  digitalWrite(RELAY,LOW); //Default state set to LOW
```
Comment (iayanpahwa) [inline src/ESP8266.cpp]: ```Doesn't works.```

**Pull request closed without merging**